integrations-and-supported-tools/fastai/notebooks/Neptune_fastai.ipynb

```diff
@@ -1,31 +1,86 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Neptune + fastai"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GG3_2HyRVY4Y",
+    "pycharm": {
+     "name": "#%% md\n"
+    },
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "This guide will show you how to:\n",
+    "\n",
+    "* Log basic metadata using `NeptuneCallback()`,\n",
+    "* Single and multi phase logging using `NeptuneCallback()`,\n",
+    "* Log model weights using `NeptuneCallback()`,\n",
+    "* Log images to a Neptune `run` using the `neptune` client."
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "WzRZxnhilGh3"
+    "id": "1tBfsYCm05rl",
+    "pycharm": {
+     "name": "#%% md\n"
+    }
    },
    "source": [
-    "# Neptune + Fastai\n",
     "## Before you start\n",
-    "**Install dependencies**"
+    "\n",
+    "This notebook example lets you try out Neptune as an anonymous user, with zero setup.\n",
+    "\n",
+    "* If you are running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example recorded to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [ Tell me more about projects](https://docs.neptune.ai/administration/projects)"
```

Comment:
Could you please fix the remaining extra spaces in this link?
`[ Tell me more` --> `[Tell me more`